libnmstate/metadata.py

```diff
@@ -198,6 +194,13 @@ def _generate_link_master_metadata(
                         )
 
 
+def _iface_to_remove(iface_state):
```

Comment:
Is this the correct name? The profiles are going to be removed and the interface only if it is a virtual interface (but not veth).

Reply:
We do not distinct at the moment (but we should) between down and absent. We look at this as if it is absent because it removes the profile, but you are right that it is problematic.

The other alternative is to call it `_is_iface_state_down_or_absent`.

Maybe it is not worth the effort and we should not do this at all.

Reply:
The other name fits. I'll leave it up to you whether to close or rename.

Reply:
If we cannot name it, it is not worth the change.
Closing.